_docs/reference/commands/istioctl.md

```diff
@@ -24,8 +24,8 @@ Available routing and traffic management configuration types:
 
 	[route-rule ingress-rule egress-rule destination-policy]
 
-See http://istio.io/docs/reference for an overview of routing rules
-and destination policies.
+See [https://istio.io/docs/reference/](https://istio.io/docs/reference/) for an
```

Comment:
Should be {{home}} instead of https://istio.io

Reply:
The text is verbatim from istioctl output with the hyperlink a bonus. but I will change it if needed.